app/templates/all_questions/cy/cof_r3w3_all_questions_cy.html

```diff
@@ -0,0 +1,1634 @@
+    1.1.3.1. Sut mae eich sefydliad yn cael ei ddosbarthu
+  </h4>
+  <p class="govuk-body">
+    Math o sefydliad (Arall)
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.3.2. Manylion cofrestru
+  </h4>
+  <p class="govuk-body">
+    Gyda pha gorff rheoleiddio y mae eich cwmni wedi'i gofrestru?
+  </p>
+  <p class="govuk-body">
+    Dewiswch un opsiwn
+  </p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li class="">
+      TÅ·'r CwmnÃ¯au
+    </li>
+    <li class="">
+      Arall
+    </li>
+  </ul>
+  <p class="govuk-body">
+    Os 'Arall', ewch i <strong>1.1.3.2.1</strong>
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.3.2. Manylion cofrestru'r cwmni
+  </h4>
+  <p class="govuk-body">
+    Rhif cofrestru'r cwmni
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.3.2.1. YnglÅ·n Ã¢'ch sefydliad
+  </h4>
+  <p class="govuk-body">
+    Manylion cofrestru
+  </p>
+  <p class="govuk-body">
+    Gyda pha gorff rheoleiddio y mae eich cwmni wedi'i gofrestru? (Arall)
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.3.3. Manylion cofrestru'r elusen
+  </h4>
+  <p class="govuk-body">
+    Charity number
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.4. Is-gwmnÃ¯au masnachu
+  </h4>
+  <p class="govuk-body">
+    A yw eich sefydliad yn is-gwmni masnachol i riant-gwmni?
+  </p>
+  <p class="govuk-body">
+    Os 'Ydy', ewch i <strong>1.1.4.1</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Nac ydy', ewch i <strong>1.1.5</strong>
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.4.1. Manylion rhiant-sefydliad
+  </h4>
+  <p class="govuk-body">
+    Enw'r rhiant-sefydliad
+  </p>
+  <p class="govuk-body">
+    Dyddiad y sefydlwyd y rhiant-sefydliad
+  </p>
+  <p class="govuk-body">
+    Er enghraifft, 27 3 2007
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.5. Cyfeiriad y sefydliad
+  </h4>
+  <p class="govuk-body">
+    Cyfeiriad y sefydliad
+  </p>
+  <p class="govuk-body">
+    A yw eich cyfeiriad gohebu yn wahanol i gyfeiriad y sefydliad?
+  </p>
+  <p class="govuk-body">
+    Os 'Ydy', ewch i <strong>1.1.5.1</strong>
+  </p>
+  <p class="govuk-body">
+    Os 'Nac ydy', ewch i <strong>1.1.6</strong>
+  </p>
+  <p class="govuk-body">
+    Gwefan a chyfryngau cymdeithasol
+  </p>
+  <p class="govuk-body">
+    Er enghraifft, cyfrifon Facebook, Instagram neu Twitter eich cwmni (os yw'n gymwys)
+  </p>
+  <p class="govuk-body">
+    Dolen 1
+  </p>
+  <p class="govuk-body">
+    Dolen 2 (dewisol)
+  </p>
+  <p class="govuk-body">
+    Dolen 3 (dewisol)
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.5.1. Cyfeiriad gohebu
+  </h4>
+  <p class="govuk-body">
+    Cyfeiriad gohebu
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.6. Ceisiadau ar y cyd
+  </h4>
+  <p class="govuk-body">
+    A yw eich cais yn gynnig ar y cyd mewn partneriaeth Ã¢ sefydliadau eraill
+  </p>
+  <p class="govuk-body">
+    Os 'Ydy', ewch i <strong>1.1.6.1</strong>
+  </p>
+  <h4 class="govuk-heading-s">
+    1.1.6.1. Manylion sefydliad partner
```

Comment:
Is there a Welsh translation for "partner"?

Reply:
Missing translation for partner

Reply:
https://dluhcdigital.atlassian.net/browse/FS-3862 ticket created